packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -148,6 +166,7 @@ class MemoryBodyState extends State<MemoryBody> with AutoDisposeMixin {
         : '';
 
     _memoryEvents ??= MemoryEventsPane();
```

Comment:
while we are here we should remove the _memoryChart and _memoryEvents local variables. Caching widgets is most likely an antipattern as it can mess up hot reload and isn't needed for performance.

Reply:
I'll remove and verify that things still look good.